src/Symfony/Component/DependencyInjection/Tests/Fixtures/includes/ProjectWithUpperAliasAndNamespaceExtension.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+
+class ProjectWithUpperAliasAndNamespaceExtension extends Extension
+{
+    public function load(array $configs, ContainerBuilder $configuration) {}
+
+    public function getNamespace()
+    {
+        return 'http://www.example.com/schema/PROJECT';
+    }
```

Comment:
the namespace isn't important for this, is it? (and namespace is not used for internal storage by the container).